server/graphql/schemaV2.graphql

```diff
@@ -944,6 +944,7 @@ enum ContributionFrequency {
   MONTHLY
   YEARLY
   ONETIME
+  RECURRING
```

Comment:
I think this value was left over, right?

Reply:
Ops, that seems to be the case. I've corrected it. Will merge ones test pass. Thanks for the reviews. 👍🏽